### PR TITLE
Correct package requirements. Fixes #18

### LINKED ISSUE #18: Correct requirements
Change to pull most recent pytest-mozwebqa version
outdated py/pytest/pytest-dist versions

### PR DESCRIPTION
**Pull request by davehunt**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,17 +1,4 @@
-# This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
-
-PyYAML==3.10
```
Comment (davehunt) [inline requirements.txt]: This is a dependency of pytest-mozwebqa, so we can leave that to manage it.

File: requirements.txt [hunk 2]
```diff
@@ -1,17 +1,4 @@
-# This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
-
-PyYAML==3.10
-UnittestZero
-certifi==0.0.8
-chardet==1.0.1
```
Comment (davehunt) [inline requirements.txt]: These two are not requirements.

File: requirements.txt [hunk 3]
```diff
@@ -1,17 +1,4 @@
-# This pulls in all the libraries needed to run Selenium tests
-# on Mozilla WebQA projects
-
-PyYAML==3.10
-UnittestZero
-certifi==0.0.8
-chardet==1.0.1
-py==1.4.9
```
Comment (davehunt) [inline requirements.txt]: This is a dependency of pytest, so let's leave that to manage it.

File: requirements.txt [hunk 4]
```diff
@@ -1,17 +1,4 @@
...
-# on Mozilla WebQA projects
-
-PyYAML==3.10
-UnittestZero
-certifi==0.0.8
-chardet==1.0.1
-py==1.4.9
-pytest==2.2.4
-pytest-mozwebqa==1.1
-pytest-xdist==1.8
```
Comment (davehunt) [inline requirements.txt]: Not a requirement of the tests.

File: requirements.txt [hunk 5]
```diff
@@ -1,17 +1,4 @@
...
-requests==1.2.0
```
Comment (davehunt) [inline requirements.txt]: This is only required via pytest-mozwebqa, so let's leave that to manage the dependency.

File: requirements.txt [hunk 6]
```diff
@@ -1,17 +1,4 @@
...
+pytest==2.7.2
+pytest-mozwebqa
 selenium
-beautifulsoup4==4.1.3
-pytz==2013b
-icalendar==3.4
--e git+https://github.com/mozilla/bidpom.git@master#egg=browserid
```
Comment (davehunt) [inline requirements.txt]: These are not currently required.

**Comment by rbillings** [comment]
Tests pass and changes look good to me.

**Pull request merged**